### PR TITLE
pages.nl/*: add Dutch translations

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.nl/windows/choco-install.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Installeer een of meerdere pakketen met Chocolatey.
+> Meer informatie: <https://chocolatey.org/docs/commands-install>.
+
+- Installeer een of meerdere spatie-gescheiden pakketten:
+
+`choco install {{pakket1 pakket2 ...}}`
+
+- Installeer pakketten van een aangepast configuratiebestand:
+
+`choco install {{pad\naar\pakketten_bastand.config}}`
```
Comment (leonvsc) [inline pages.nl/windows/choco-install.md:12]:
```suggestion
`choco install {{pad\naar\pakketten_bestand.config}}`
```

File: pages.nl/windows/choco-push.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# choco-push
+
+> Push een gecompileerd NuGet pakket (`nupkg`) naar een pakketfeed.
+> Meer informatie: <https://docs.chocolatey.org/en-us/create/commands/push>.
+
+- Push een gecompileerd `nupkg` naar de gespecificeerde feed:
+
+`choco push --source {{https://push.chocolatey.org/}}`
+
+- Push een gecompileerd `nupkg` naar de gespecificeerde feed met een timeout in seconden (default is 2700):
```
Comment (leonvsc) [inline pages.nl/windows/choco-push.md:10]:
```suggestion
- Push een gecompileerd `nupkg` naar de gespecificeerde feed met een timeout in seconden (standaard is 2700):
```

File: pages.nl/windows/choco-uninstall.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> Verwijder een of meerdere pakketen met Chocolatey.
+> Meer informatie: <https://chocolatey.org/docs/commands-uninstall>.
+
+- Verwijder een of meerdere spatie-gescheiden pakketten:
+
+`choco uninstall {{pakket1 pakket2 ...}}`
+
+- Verwijder een specifieke versie van een pakket:
+
+`choco uninstall {{pakket}} --version {{version}}`
```
Comment (leonvsc) [inline pages.nl/windows/choco-uninstall.md:12]:
```suggestion
`choco uninstall {{pakket}} --version {{versie}}`
```

File: pages.nl/windows/choco.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# choco
+
+> De Chocolatey pakket manager.
+> Sommige subcommando's zoals `choco install` hebben hun eigen gebruiksdocumentatie.
+> Meer informatie: <https://chocolatey.org>.
+
+- Voer een Chocolatey commando uit:
+
+`choco {{command}}`
```
Comment (leonvsc) [inline pages.nl/windows/choco.md:9]:
```suggestion
`choco {{commando}}`
```

File: pages.nl/windows/choco.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- Toon algemene hulp:
+
+`choco -?`
+
+- Toon help voor een specifiek commando:
+
+`choco {{command}} -?`
```
Comment (leonvsc) [inline pages.nl/windows/choco.md:17]:
```suggestion
`choco {{commando}} -?`
```

File: pages.nl/windows/select-string.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Select-String
+
+> Vindt tekst in string en bestanden in PowerShell.
+> Dit commando kan alleen gebruikt worden via PowerShell.
+> Je kan `Select-String` gebruiken zoaks `grep` in UNIX of `findstr.exe` in Windows.
```
Comment (leonvsc) [inline pages.nl/windows/select-string.md:5]:
```suggestion
> Je kan `Select-String` gebruiken zoals `grep` in UNIX of `findstr.exe` in Windows.
```

File: pages.nl/windows/where-object.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Where-Object
+
+> Selecteert objecten uit een verzameling op basis van hun eigenschapswaarden.
+> Dit commando kan alleen gebruikt worden via PowerShell.
+> Meer informatie: <https://learn.microsoft.com/powershell/module/microsoft.powershell.core/where-object>.
+
+- Filter aliassen op naam:
+
+`Get-Alias | Where-Object -{{Property}} {{Name}} -{{eq}} {{name}}`
```
Comment (leonvsc) [inline pages.nl/windows/where-object.md:9]:
```suggestion
`Get-Alias | Where-Object -{{Propertie}} {{Naam}} -{{eq}} {{naam}}`
```

**Review by leonvsc — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have a few suggestions. Mostly some spelling mistakes or some forgotten translations.

**Review by leonvsc — APPROVED** [review]

**Pull request merged**